e2e/solve/testutil.go

```diff
@@ -0,0 +1,164 @@
+package solve
```

Comment:
I would rename `testutil.go` to `order_tracker.go` to better represent what this file contains.

Reply:
there is more in there than that

Reply:
ill put order tarcket into a separate file